specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.0/TextAnalytics.json

```diff
@@ -663,7 +663,8 @@
         "sentiment",
         "confidenceScores",
         "offset",
-        "length"
+        "length",
+        "text"
```

Comment:
You're introducing a breaking change in a stable API version.  Can you please ACK that this is required?

Reply:
For context: https://teams.microsoft.com/l/message/19:d6ff4003f5c848a2a2d6dcdfc0ebd497@thread.skype/1594933648943?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1594933648943&teamName=Azure%20SDK&channelName=Service%20-%20Text%20Analytics&createdTime=1594933648943

It is a mistake in the old swagger file.